include/eosio/ship_protocol.hpp

```diff
@@ -266,7 +266,46 @@ namespace eosio { namespace ship_protocol {
    EOSIO_REFLECT(action_trace_v1, action_ordinal, creator_action_ordinal, receipt, receiver, act, context_free, elapsed,
                  console, account_ram_deltas, except, error_code, return_value)
 
-   using action_trace = std::variant<action_trace_v0, action_trace_v1>;
+   struct sync_call_trace {
+      eosio::varuint32             ordinal;
+      eosio::varuint32             sender_ordinal;
+      eosio::name                  sender;
+      eosio::name                  receiver;
+      uint64_t                     flags;
+      eosio::input_stream          data;
+      int64_t                      elapsed;
+      std::string                  console;
+      std::optional<std::string>   except;
+      std::optional<uint64_t>      error_code;
+      int64_t                      return_value_size_or_error_id;
+      eosio::input_stream          return_value;
+   };
```

Comment:
All the other structs in this file have default initializers so these should too

Reply:
https://github.com/AntelopeIO/abieos/pull/42/commits/b6c2c866b5b06dafa1ce3e41dca45cbd8cab1e64